league/item.py

```diff
@@ -0,0 +1,32 @@
+from .stats import Stats
+
+class Item:
+    def __init__(self, name: str, level: int, stat_buff: Stats):
+        self.name = name
+        if level < 1:
+            raise ValueError("Level can't be negative.")
+        self.level = level
+        self.stat_buff = stat_buff
+
+    def get_name(self) -> str:
+        return self.name
+
+    def set_name(self, name: str) -> None:
```

Comment:
Don't think we need the setters for any of the attributes, just leave the getters.